application/src/test/java/org/opentripplanner/transit/speed_test/SpeedTest.java

```diff
@@ -191,7 +193,12 @@ public void runTest() {
       }
     }
 
+    measureTransferCacheComputation();
```

Comment:
The SpeedTest is still the most important tool we have to measure the performance of Raptor, so I am not sure this should be done in the SpeedTest, why is this not in a separate test? We should probably discuss it.